php/src/php/Util.cpp

```diff
@@ -483,23 +509,39 @@ convertLocalException(const Ice::LocalException& ex, zval* zex)
             zval_ptr_dtor(&id);
             return false;
         }
+#if PHP_VERSION_ID >= 80000
```

Comment:
maybe you should add a helper for call `zend_update_property` so we don't have this macro everywhere